pkg/gui/gui.go

```diff
@@ -203,19 +199,6 @@ type GuiRepoState struct {
 	CurrentPopupOpts *types.CreatePopupPanelOpts
 }
 
-type Controllers struct {
```

Comment:
removing this struct because we shouldn't need to keep references to controllers: they're just there to be attached to contexts